lib/3scale/backend/errors.rb

```diff
@@ -333,5 +333,11 @@ def initialize(service_id, msg)
         super "Delete stats job context validation error. Service: #{service_id}. Error: #{msg}"
       end
     end
+
+    class EndUsersNoLongerSupported < BadRequest
+      def initialize
+        super 'End-users are no longer supported'.freeze
```

Comment:
`"[...], do not specify the user_id parameter"`?

Reply:
Done :+1: